backend/app/api/v1/commons/quay.py

```diff
@@ -27,7 +27,7 @@ async def getData(start_datetime: date, end_datetime: date, size, offset, config
     tasks = [item['_source'] for item in response['data']]
     jobs = pd.json_normalize(tasks)
     if len(jobs) == 0:
-        return jobs
+        return ({'data':jbs, 'total': 0})
```

Comment:
`jbs` => `jobs`

Reply:
Actually, 

`  cleanJobs = jobs[jobs["platform"] != ""]  
    jbs = cleanJobs `
    
    It's not typo :smirk: